db/agents.json

```diff
@@ -1369,14 +1438,38 @@
       ]
     },
     {
-      "regex": "^Podcasts",
+      "regex": "^(Podcasts|Podcast’ler|Podcast|Podcaster|Podcasti|Podcastit|Podcastok|Podcasturi|Podcasty|Podkaster|Balados|Подкасти|Подкасты|פודקאסטים|البودكاست|पॉडकास्ट|พ็อดคาสท์|播客|팟캐스트)\\/.+(x86_64)",
+      "name": "Apple Podcasts",
+      "type": "Desktop App",
+      "os": "macOS",
+      "examples": [
+        "Podcasts/1425.5 CFNetwork/1121.1.2 Darwin/19.3.0 (x86_64)",
+        "Подкасты/1425.5 CFNetwork/1121.1.2 Darwin/19.2.0 (x86_64)",
+        "Podcast/1425.5 CFNetwork/1121.1.2 Darwin/19.2.0 (x86_64)",
+        "Podkaster/1425.5 CFNetwork/1121.1.2 Darwin/19.2.0 (x86_64)",
+        "Podcasty/1425.5 CFNetwork/1121.1.2 Darwin/19.2.0 (x86_64)",
+        "Balados/1425.5 CFNetwork/1121.1.2 Darwin/19.2.0 (x86_64)",
+        "พ็อดคาสท์/1425.5 CFNetwork/1121.1.2 Darwin/19.3.0 (x86_64)",
+        "Podcast’ler/1425.5 CFNetwork/1121.1.2 Darwin/19.2.0 (x86_64)",
+        "%ED%8C%9F%EC%BA%90%EC%8A%A4%ED%8A%B8/1425.5 CFNetwork/1121.1.2 Darwin/19.3.0 (x86_64)"
```

Comment:
In our usage (a node + express app) these headers come out URL encoded.  Some chance it's actually the requester doing this encoding.  So ... belt and suspenders, I'm testing both here.